tests/integration/targets/sudoers/tasks/main.yml

```diff
@@ -145,6 +147,42 @@
   register: edge_case_1
 
 
+- name: Attempt command without full path to executable, but disabling validation
+  community.general.sudoers:
+    name: edge-case-2
+    state: present
+    user: alice
+    commands: systemctl
+    validation: absent
+  register: edge_case_2
+
+- name: find visudo
+  command:
+    cmd: which visudo
+  ignore_errors: true
```

Comment:
Why have `ignore_errors: true` if in the next task you explicitly verify that the task didn't fail?

Reply:
Yeah, that's a good point!